syslog-ng/Dockerfile

```diff
@@ -6,8 +6,8 @@ RUN apt-get update -qq && apt-get install -y \
     wget \
     gnupg2
 
-RUN wget -qO - https://ose-repo.syslog-ng.com/apt/syslog-ng-ose-pub.asc | apt-key add - && \
-  echo "deb https://ose-repo.syslog-ng.com/apt/ stable debian-testing" | tee --append /etc/apt/sources.list.d/syslog-ng-ose.list
+RUN wget -qO - https://ose-repo.syslog-ng.com/apt/syslog-ng-ose-pub.asc | gpg --dearmor > /usr/share/keyrings/ose-repo-archive-keyring.gpg && \
```

Comment:
I think we should install ca-certificates explicitly. I was able to reproduce an APT fetch error once in a docker image without it.